README.md

```diff
@@ -123,10 +132,106 @@ To install pgai from source on a PostgreSQL server:
    The `CASCADE` automatically installs `pgvector` and `plpython3u` extensions.
 
 ### Use pgai
- 
-Now, use pgai to integrate AI from [Ollama](./docs/ollama.md), [OpenAI](./docs/openai.md), 
-[Anthropic](./docs/anthropic.md), and [Cohere](./docs/cohere.md). Learn how to [moderate](./docs/moderate.md) and
-[embed](./docs/delayed_embed.md) content directly in the database using triggers and background jobs.
+
+The main features in pgai are:
+
+**Working with embeddings generated from your data:**
+* [Automatically create and sync vector embeddings for your data](#automatically-create-and-sync-llm-embeddings-for-your-data)
+* [Search your data using vector and semantic search](#search-your-data-using-vector-and-semantic-search)
+* [Implement Retrieval Augmented Generation inside a single SQL statement](#implement-retrieval-augmented-generation-inside-a-single-sql-statement) 
+
+**Leverage LLMs for data processing tasks:**
+You can use pgai to integrate AI from [Ollama](./docs/ollama.md), [OpenAI](./docs/openai.md), 
+[Anthropic](./docs/anthropic.md), and [Cohere](./docs/cohere.md). Learn how to [moderate](./docs/moderate.md) and content directly in the database using triggers and background jobs.
+
+
+### Automatically create and sync LLM embeddings for your data
+
+The [pgvector](https://github.com/pgvector/pgvector) and 
+[pgvectorscale](https://github.com/timescale/pgvectorscale) extensions allow you
+to store vector embeddings in your database and perform fast and efficient
+vector search.  The [pgai Vectorizer](/docs/vectorizer.md) builds on top of
+these extensions to automatically create and synchronizes embeddings for any
+text data in your database.
+
+With one line of code, you can define a vectorizer that creates embeddings for data in a table:
```

Comment:
Do you think we should mention that you still need something to process those embeddings? In this case to run the Vectorizer Worker.

I'm concern about being a little disingenuous with "just one query", then you keep reading and find a "but then you have to run this other thing".

Reply:
I mean, this is true for cloud, but not for self hosted. I don't have any ideas for this to be fair, so it might be fine as it is.

Reply:
I'm going to leave for now